radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -282,3 +282,36 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+// Author - Atharva (iya789)
+// May 7, 2022
+
+/**
+ *Resets microcontroller of the antenna
+ *
+ * @return 0 for success, non-zero for failure. See hal/errors.h for details.
+ */
+int antennaReset(void){
+
+	//command code for reset: 10101010 (section 6.2 of antenna user manual)
+
+	// form SSI library
+
+	int errorA = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideA );  //reset on side A
+	int errorB = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideB );  //reset on side B
```

Comment:
- Try and manage your whitespace a little better. Thats 7 lines for 2 very similar statements. 
- numeric codes should always be represented using definitions and shouldn't be in documentations unless extremely necessary. I think it was nice to mention the manual though.
- TBH no code here needs comments but I would simplify it to "// Reset both side A/B antennas. See section 6.2 of Antenna System User Manual"
- There should be spaces after commas in this context and no space at the end of the arguments. This is standard not only in QT but in most style guides and languages.

Reply:
Agreed with everything above. I would also advise against "trailing comments" (comments on the same line as a statement, as in line 300), especially when the statement is long. A single comment on its own line as Austin mentioned would suffice

Reply:
EOL comments not fixed on new commit.